definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-prod.json

```diff
@@ -47,5 +47,12 @@
     "CaseEventID": "EnterAOSOffline_CoRespondent",
     "UserRole": "caseworker-divorce-courtadmin_beta",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalRejection",
```

Comment:
Is this an existing event @Dizrupto, or did you create it for this story?

Reply:
existing one :)